physics/src/modules/DamageHealingModule/ConstantHealing.cpp

```diff
@@ -51,6 +51,12 @@ void ConstantHealing::update(const TimestepTime& tstep)
 
 void ConstantHealing::updateElement(size_t i, const TimestepTime& tstep)
 {
+    // No ice, no healing
+    if (cice[i] == 0.) {
```

Comment:
Might it be worth doing an approximate check? e.g.,
```suggestion
    double tol = 1e-10;
    if (abs(cice[i]) < tol) {
```
Otherwise unexpected behaviour might result when `cice` is effectively (but not quite) zero.

Reply:
Yes, good point. I'll use IceMinima::c(), which should be used for this purpose throughout the model. In particular, the thermodynamics melts all ice in the grid cell when the concentration drops below this limit.